src/app/drive/components/ShareDialog/ShareDialog.tsx

```diff
@@ -246,11 +246,19 @@ const ShareDialog = (props: ShareDialogProps): JSX.Element => {
 
     const isItemNotSharedYet = !isAdvancedShareItem(itemToShare?.item) && !itemToShare.item.sharings?.length;
 
+    const sharingInfo = await shareService.getSharingInfo(itemId, itemType).catch((error) => {
+      if (error?.code === 'SHARING_NOT_FOUND') {
```

Comment:
This ‘code’ field that returns the call is correct?

Reply:
Mmm no, it seems to be used only when calculating the size of the shared folder... removing it.